src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -577,6 +577,10 @@
   <data name="Nav_OpenJSON.Content" xml:space="preserve">
     <value>Open JSON file</value>
     <comment>Header for a menu item. This opens the JSON file that is used to log the app's settings.</comment>
+  </data>

```

Comment:
Also, make sure you use spaces here, not tabs.

If you want, you can also edit this file in Visual Studio. It gives you a table to work with, and always formats it correctly.